Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/Bookmarks/BookmarkDetailsView.swift

```diff
@@ -88,6 +94,45 @@ class BookmarkDetailsView: AddEditHeaderView, BookmarkFormFieldsProtocol {
     // MARK: - Delegate actions
     
     @objc func textFieldDidChange(_ textField: UITextField) {
-        delegate?.correctValues(validationPassed: validateFields())
+        delegate?.correctValues(validationPassed: validateFields() || validateCodeFields())
+    }
+    
+    private func validateTitle(_ title: String?) -> Bool {
+        guard let title = title else { return false }
+        return !title.isEmpty
+    }
+    
+    private func validateCodeFields() -> Bool {
```

Comment:
this could use a unit test

Reply:
Fixed. Added a unit test for Bookmarklets.